src/stmt.cpp

```diff
@@ -2269,30 +2269,30 @@ void ForeachActiveStmt::Print(Indent &indent) const {
 
     indent.setNextLabel("iter symbol");
     indent.Print();
-    if (sym != NULL) {
+    if (sym != nullptr) {
         printf("%s", sym->name.c_str());
-        if (sym->type != NULL)
+        if (sym->type != nullptr)
             printf(" %s", sym->type->GetString().c_str());
     } else {
-        printf("NULL");
+        printf("nullptr");
```

Comment:
Here and in some other Print cases below, should we keep NULL as well?

Reply:
These are AST dump output / debug prints. Both are fine, but probably `NULL` is better as it's all capital word, which sticks out in the output drawing attention to it (and this is good, as it's an error typically).